src/main/java/spike/commands/InputType.java

```diff
@@ -0,0 +1,19 @@
+package spike.commands;
+
+/**
+ * Enumerates the different types of inputs that the user can provide.
+ */
+public enum InputType {
+    BYE,
+    LIST,
+    LISTBYDATE,
+    MARK,
+    UNMARK,
+    TODO,
+    DEADLINE,
+    EVENT,
+    TASK,
+    DELETE,
+    FIND,
+    ERROR
+}
```

Comment:
Great job using enums for your commands! It makes it easier for others to understand what commands are there.